ingest.py

```
@@ -0,0 +1,65 @@
+import pandas as pd
+
+def process_docx(file_path):
+    """Lee un archivo .docx y extrae el texto de los párrafos."""
+    try:
+        doc = docx.Document(file_path)
+        return "\n".join([para.text for para in doc.paragraphs if para.text.strip()])
+    except Exception as e:
+        print(f"Error procesando {file_path}: {e}")
+        return ""
+
+def process_pdf(file_path):
```

Ruff (0.12.2)

Missing return type annotation for public function `process_pdf`

(ANN201)